pkgs/top-level/all-packages.nix

```diff
@@ -25742,7 +25742,7 @@ with pkgs;
 
   x265 = callPackage ../development/libraries/x265 { };
 
-  xandikos = callPackage ../servers/xandikos { };
+  xandikos = python3.pkgs.callPackage ../servers/xandikos { };
```

Comment:
`callPackage` should be kept for [RFC 140](https://github.com/NixOS/rfcs/blob/master/rfcs/0140-simple-package-paths.md).